app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class VotesController < ApplicationController
+
+  def create
+    @review = Review.find(params[:review_id])
+    @review.votes.where(:user_id => current_user.id).destroy_all
```

Comment:
You don't need to delete the records first. You can try to see if a user has voted on this review and then update it or create a new one:

```ruby
def create
  review = Review.find(params[:review_id])
  vote = review.votes.find_or_initialize_by(user: current_user)
  vote.assign_attributes(vote_params)
  vote.save
  redirect_to buddy_path(review.buddy)
end
```